docs/contribute.rst

```diff
@@ -140,18 +139,16 @@ label. The reported issue …
     - *My builds stopped working. Please help!*
       Ask for a link to the build log and for which project is affected.
 
-… is a valid enhancement proposal
-    If the ticket contains an enhancement proposal that aligns with the goals
-    of Read the Docs, then add the label **Enhancement**. If the proposal
+… is a valid feature proposal
```

Comment:
I think `enhancement` is now `feature`, but we also have `Improvement` (that's why I removed some examples here)